src/python/circllhist/__init__.py

```diff
@@ -0,0 +1,77 @@
+import circllhist.ffi as ffi
+
+
+class circllbin:
+    def __init__(self, b):
+        self.b = b
+
+    @classmethod
+    def new(cls):
+        return cls(ffi.ffi.new("hist_bucket_t*"))
```

Comment:
This creates an invalid object, as `hist_bucket_t` is a value type. Should either call `from_number` with a sensible default value,  or get rid of `new`.

Reply:
I see, you want a `hist_bucket_t` allocated on the stack. I am not sure how to do that...
Anyways the methods is not needed. I am just removing it. We have `from_number()`.